src/ansys_sphinx_theme/__init__.py

```diff
@@ -464,6 +470,105 @@ def configure_theme_logo(app: Sphinx):
         theme_options["logo"] = theme_options.get("logo")
 
 
+def convert_pdf_to_png(pdf_path: pathlib.Path, output_dir: pathlib.Path, output_png: str):
+    """
+    Convert PDF to PNG images.
+
+    Parameters
+    ----------
+    pdf_path : pathlib.Path
+        Path to the PDF file.
+    output_dir : pathlib.Path
+        Path to the output directory.
+    output_png : str
+        Name of the output PNG file.
+    """
+    try:
+        doc = pymupdf.open(pdf_path)  # open document
+        for page in doc:
+            pix = page.get_pixmap()
+            pix.save(os.path.join(output_dir, output_png))
+    except Exception as e:
```

Comment:
Let's avoid capturing generic exceptions. Can we narrow which exceptions are involved here?

Reply:
Sure, adding that.

Reply:
added in [1544c20](https://github.com/ansys/ansys-sphinx-theme/pull/428/commits/1544c20529ed0969baca26555a6471a42bda43cd)